### PR TITLE
Add diagnostic endpoint

### PR DESCRIPTION
**Pull request by ostapneko**

### CODE REVIEW

**Comment by ostapneko** [comment]
@tomverran @afiore

**Comment by ostapneko** [comment]
image

image

File: frontend/app/services/SubscriptionService.scala [hunk 1]
```diff
@@ -108,11 +108,15 @@ class SubscriptionService(val zuoraSoapClient: soap.ClientWithFeatureSupplier,
...
-    for {
-      zuoraCatalog <- zuoraRestClient.productCatalog
-    } yield MembershipCatalog.unsafeFromZuora(ratePlanIds)(zuoraCatalog.products.flatMap(_.productRatePlans))
+    productRatePlans.map(MembershipCatalog.unsafeFromZuora(ratePlanIds))
   )
 
+  def productRatePlans: Future[Seq[rest.ProductRatePlan]] =
+     zuoraRestClient.productCatalog.map(_.products.flatMap(_.productRatePlans))
+
+  def makeMembershipCatalog(): Future[MembershipCatalog.Val[MembershipCatalog]] =
```
Comment (afiore) [inline frontend/app/services/SubscriptionService.scala]: Not sure about this name. `makeMembershipCatalog` suggest some side effects, like persisting a record or similar...

**Comment by tomverran** [comment]
looks okay to me if you can get travis to pass it :+1:

**Comment by ostapneko** [comment]
Seen in Prod:

image

**Pull request merged**